spec/helpers/money_helper_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe MoneyHelper, type: :helper do
+  let(:bank_account_gbp) { create(:bank_account, currency: 'GBP') }
+  let(:bank_account_eur) { create(:bank_account, currency: 'EUR') }
+  let(:bank_account_usd) { create(:bank_account, currency: 'USD') }
+  let(:bank_account_rsa) { create(:bank_account, currency: 'RSA') }
+
+  describe '#display_value_with_symbol' do
+    it 'return value with pound symbol' do
+      expect(display_value_with_symbol(bank_account_gbp.balance, bank_account_gbp.currency)).to eq "£#{bank_account_gbp.balance}"
```

Comment:
**[improvement]** Given we're just testing an helper, I don't believe there's any need here to create records in the database as the helper itself just needs an integer and a string as arguments.

**[not a blocker/nit pick]** In terms of readability it would be clearer to have something like the following instead:

```ruby
describe '#display_value_with_symbol' do
  let(:balance) { 325.42 }

  subject(:humanized_balance) { display_value_with_symbol(balance, currency) }

  context 'when the currency is Euro' do
    let(:currency) { 'EUR' }

    it { is_expected.to eq("€#{balance}") } 
  end
end
```